src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -766,19 +766,24 @@ function relativeTimeToDate(text, now = new Date()) {
 // If it is not a valid constraint but it could be a valid something
 // else, return CannotTransform.
 // inArray is whether this is an array field.
-function transformConstraint(constraint, field, count = false) {
-  const inArray = field && field.type && field.type === 'Array';
+function transformConstraint(constraint, restKey, field, count = false) {
```

Comment:
Can you rename `restKey` to `key`?

Reply:
There is actually a `key` variable in the for loop that concerns contraint.